language/de-DE/tpl_cassiopeia.ini

```diff
@@ -7,6 +7,7 @@
 CASSIOPEIA="Cassiopeia Frontend Template"
 TPL_CASSIOPEIA_BACKTOTOP="Zurück nach oben"
 TPL_CASSIOPEIA_BACKTOTOP_LABEL="„Zurück nach oben“ Link"
+TPL_CASSIOPEIA_BRAND_LABEL="Logo anzeigen"
```

Comment:
```suggestion
TPL_CASSIOPEIA_BRAND_LABEL="Logo (Brand)"
```

Reply:
Vielleicht sogar besser Marke als Übersetzung von Brand?

Reply:
Ich bezweifle, dass die Mehrzahl der Joomla-User die Werbe/Marketing-Begriffe Brand, Marke und Logo klar unterscheiden können.
Falls unbedingt ein dt. Begriff verwendet werden soll vielleicht "Markenzeichen"

Andererseits, falls ich den engl. [Issue](https://github.com/joomla/joomla-cms/pull/33751) richtig verstehe, wollte Christiane damit eine Möglichkeit schaffen, um das Logo auszublenden.
Warum in der Core jetzt nur "Brand" verwendet wird, erschließt sich mir nicht ganz.

Reply:
Muss ich mir im Template mal ansehen wie es dort aussieht.

Reply:
Hab mir den PR und die ganze Diskussion jetzt mal gegeben.
Tatsächlich geht es um das Ausblenden der Position im Code, wenn man kein Logo etc verwendet.
Von daher würde ich das schlicht auf Brand lassen, da es mehr ist als Logo.